all-requirements.txt

```diff
@@ -10,3 +10,6 @@ tornado==4.2
 
 # For simpl.middleware.cors
 WebOb==1.4.1
+
+# MongoDB
+pymongo==2.8.1
```

Comment:
I've added support for pymongo 3.0.3 (and still works with 2.8.1). The code still applies; there's no automatic reconnect, so we have an implementation for that, although I've used a more modern one that addresses issues we had with cursors.. Much of the text search ad cursor code enables pagination and works with rest.py. That's all still valid.